addons/im_livechat/views/im_livechat_channel_templates.xml

```diff
@@ -189,7 +189,7 @@
             <t t-translation="off">
 
             <t t-if="not info">
-                <t t-set="info" t-value="request.env['im_livechat.channel'].get_livechat_info(channel)"/>
+                <t t-set="info" t-value="request.env['im_livechat.channel'].sudo().browse(channel).get_livechat_info()"/>
```

Comment:
Zizisse louche. Can't we perform some check in get_livechat_info and sudo its computation once we are sure access is granted ?

Reply:
fix with the proxy method on website that we discussed. Changes are included in the REF commit (1st one).